student.rb

```diff
@@ -1,4 +1,5 @@
 require_relative 'person'
+
 class Student < Person
   def initialize(age, classroom, name: 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
```

Comment:
- [ ] I noticed you have not set the reader/getter for the classroom, it is needed because you also need a method that you should also create which is to serve as the 'belongs-to' side (a student belongs to a classroom). This is needed so that when setting the classroom for a student it also adds it to the classrooms' students. It can look like this:
```
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```